vmm/sandbox/src/stratovirt/mod.rs

```diff
@@ -69,29 +70,22 @@ pub(crate) const STRATOVIRT_START_TIMEOUT_IN_SEC: u64 = 10;
 // but skip all the fields serialization.
 #[derive(Default, Serialize, Deserialize)]
 pub struct StratoVirtVM {
-    #[serde(skip)]
     id: String,
-    #[serde(skip)]
     config: StratoVirtConfig,
     #[serde(skip)]
     devices: Vec<Box<dyn StratoVirtDevice + Sync + Send>>,
     #[serde(skip)]
     hot_attached_devices: Vec<Box<dyn StratoVirtHotAttachable + Sync + Send>>,
-    #[serde(skip)]
     fds: Vec<RawFd>,
-    #[serde(skip)]
     console_socket: String,
-    #[serde(skip)]
     agent_socket: String,
-    #[serde(skip)]
     netns: String,
     #[serde(skip)]
     block_driver: BlockDriver,
     #[serde(skip)]
     wait_chan: Option<Receiver<(u32, i128)>>,
     #[serde(skip)]
     client: Option<QmpClient>,
-    #[serde(skip)]
     virtiofs_daemon: Option<VirtiofsDaemon>,
     #[serde(skip)]
     pcie_root_bus: PcieRootBus,
```

Comment:
We need to check carefully of these fields, if it is skipped from serialization, it should also be rebuild in some other way, otherwise operations to the VM may failed. the this pcie_root_bus for example, if the value is not recovered, how can we hot attach a new device to the vm, if a new container is created in the sandbox?

Reply:
fixed